test/test_time.cpp

```diff
@@ -287,6 +287,37 @@ TEST_F(TestTimeFixture, test_rcutils_time_point_value_as_nanoseconds_string) {
   EXPECT_STREQ("-0000000000000000100", buffer);
 }
 
+// Tests the rcutils_time_point_value_as_date_string() function.
+TEST_F(TestTimeFixture, test_rcutils_time_point_value_as_date_string) {
+  rcutils_ret_t ret;
+  rcutils_time_point_value_t timepoint;
+  char buffer[256] = "";
+
```

Comment:
> So it looks like the runner needs to add the TZ variable and ideally set it to UTC. I don't know how to do that or if I even have the rights to change the environment variables of the runner. It would be great to get this useful feature into ROS. But I think somebody with more knowledge and maybe access needs to help us get this going.

how about this?

```suggestion

  // change timezone into UTC for the test process
  ASSERT_TRUE(rcutils_set_env("TZ", "UTC"));
```

Reply:
I'm going to suggest we do something different, and just use `rcutils_system_time_now` as the timepoint.  We won't be able to compare against a known string, but we can at least validate that it has the parts we expect (i.e. YYY-MM-DD HH:MM::SS.ss), and it sidesteps this whole issue (which probably won't work on Windows anyway).

Reply:
I like this idea, I will get to it soonish